boards/atmega256rfr2-xpro/Makefile.include

```diff
@@ -1,7 +1,7 @@
 # configure the terminal program
 PORT_LINUX ?= /dev/ttyACM0
 PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.usbmodem*)))
-BAUD ?= 9600
+BAUD ?= 115200
```

Comment:
This should match `STDIO_UART_BAUDRATE` in `board.h`

Reply:
It does. Since STDIO_UART_BAUDRATE is not overridden anymore by the board, it uses the default one from `stdio_uart.h`
https://github.com/RIOT-OS/RIOT/blob/d02ed3a3ff9fea82b24425e4560fd6699b9c2c77/sys/include/stdio_uart.h#L51